hardhat.config.ts

```diff
@@ -1,9 +1,11 @@
-import { HardhatUserConfig } from "hardhat/config";
-import "@nomicfoundation/hardhat-toolbox";
-import "@typechain/hardhat";
-import "@nomicfoundation/hardhat-ethers";
-import "@nomicfoundation/hardhat-chai-matchers";
-import "dotenv/config";
+/** @format */
```

Comment:
What's this for? Can you please use the default prettier configuration so we all have same file formatting?

Otherwise it will be very messy on PRs because of the different prettiers 😄 

Otherwise, looks good